src/cli/Cli.ts

```diff
@@ -933,16 +934,36 @@ export class Cli {
     else {
       console.log();
     }
+
+    // Restart the spinner if it was running before the log
+    /* c8 ignore next 3 */
+    if (spinnerSpinning) {
+      cli.spinner.start();
+    }
   }
 
   private static async error(message?: any, ...optionalParams: any[]): Promise<void> {
-    const errorOutput: string = await Cli.getInstance().getSettingWithDefaultValue(settingsNames.errorOutput, 'stderr');
+    const cli = Cli.getInstance();
+    const spinnerSpinning = cli.spinner.isSpinning;
+
+    /* c8 ignore next 3 */
+    if (spinnerSpinning) {
+      cli.spinner.stop();
+    }
+
+    const errorOutput: string = Cli.getInstance().getSettingWithDefaultValue(settingsNames.errorOutput, 'stderr');
```

Comment:
Since we have CLI's instance assigned to a const, we can simplify this by using `cli.getSetting...`